framework/LoopMeUnitedSDK/Core/Internal/Builder/LoopMeORTBTools.m

```diff
@@ -107,6 +107,7 @@ - (NSData *)makeRequestBody {
             @"make": @"Apple",
             @"os": @"iOS",
             @"js": @1,
+            @"pxratio": @([LoopMeIdentityProvider calculatePxratioOfScreen]),
```

Comment:
This needs to be formatted with `NSNumberFormatter`.

// Assuming `LoopMeIdentityProvider` is already imported and calculatePxratioOfScreen is available

float pxratio = [LoopMeIdentityProvider calculatePxratioOfScreen];
NSNumber *pxratioNumber = @(pxratio);

// Create a number formatter
NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
[numberFormatter setNumberStyle:NSNumberFormatterDecimalStyle];
[numberFormatter setMinimumFractionDigits:3];
[numberFormatter setMaximumFractionDigits:3];

// Format the number as a string
NSString *formattedPxratio = [numberFormatter stringFromNumber:pxratioNumber];

// Print the formatted string
NSLog(@"Formatted pxratio: %@", formattedPxratio);